docs/user-guide/kubectl/kubectl_describe.md

```diff
@@ -18,11 +18,6 @@
 If you are using a released version of Kubernetes, you should
 refer to the docs that go with that version.
 
-<!-- TAG RELEASE_LINK, added by the munger automatically -->
```

Comment:
This removal is what hack/verify-generated-docs.sh is complaining about. Try make clean and rerun?